website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles"
+description: |-
+  Get information about iam_trusted_profiles
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles
+
+Provides a read-only data source for iam_trusted_profiles. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
reference link about iam trusted profile would help user here. Something like this

For more information, about trusted profile, see [creating trusted profiles](https://cloud.ibm.com/docs/account?topic=account-create-trusted-profile&interface=ui).